src/js/components/recipientLanding/table/RecipientLandingTable.jsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * RecipientLandingTable.jsx
+ * Created by David Trinh 7/3/18
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import StateLandingTableSorter from 'components/stateLanding/table/StateLandingTableSorter';
+import RecipientLinkCell from './RecipientLinkCell';
+
+const propTypes = {
+    loading: PropTypes.bool,
+    error: PropTypes.bool,
+    results: PropTypes.array,
+    setSort: PropTypes.func,
+    sortField: PropTypes.string,
+    order: PropTypes.object,
+    searchString: PropTypes.string
+};
+
+const RecipientLandingTable = (props) => {
+    const hideBody = props.loading || props.error || props.results.length === 0 ? 'recipient-list__body_hide' : '';
+
+    const body = props.results.map((row) => (
+        <tr
+            key={row.id}
+            className="recipient-list__body-row">
+            <RecipientLinkCell
+                id={row.id}
+                type={row.recipientLevel}
+                name={row.name}
+                searchString={props.searchString} />
+            <td className="recipient-list__body-cell recipient-list__body-cell_left">
+                {row.duns}
+            </td>
+            <td className="recipient-list__body-cell recipient-list__body-cell_center">
+                {row.amount}
+            </td>
+        </tr>
+    ));
+
+    let message = null;
+    if (props.error) {
```

Comment:
API error/ loading messages are already handled in `RecipientLandingResultsSection`. "No results" messages can stay here.